frontend/packages/topology/src/behavior/withRemoveConnector.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
 import { observer } from 'mobx-react';
+import { referenceFor, modelFor } from '@console/internal/module/k8s';
+import { useAccessReview } from '@console/internal/components/utils';
```

Comment:
The topology package shouldn't have dependencies on other `@console` packages (with the possible exception of shared, and then only with pure utils).

Reply:
Even shared we want to avoid because we eventually want to move it to PF.
There is a small dependency on popper because recently I moved that code and didn't want to duplicate.

Reply:
@jeff-phillips-18 @christianvogt Added the access check in BaseEdge component